modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-installing-oadp-rosa-sts_{context}"]
+= OADP on ROSA with STS
+
+You can employ OpenShift API for Data Protection (OADP) with Red Hat OpenShift Service on AWS (ROSA) clusters to backup and restore application data. A ROSA deployment of OpenShift is configured specifically for AWS services. AWS STS is a global web service that provides short-term credentials for IAM or federated users. ROSA with STS is the recommended credential mode for ROSA clusters.
+
+[The following sentence was in the source. Is it OK? is it too vague?] Not all of the features and options provided by OADP are recommended or supported while using ROSA clusters with OADP.
+
+[IMPORTANT]
+====
+Disable Restic for OADP in ROSA environments because Restic is not supported in such environments. Instead use the native snapshots workflow to backup volumes.
+====
+
+.Prerequisites
+
+* A ROSA OpenShift Cluster with the required access and tokens.
+* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/backup_and_restore/application-backup-and-restore#oadp-creating-default-secret_installing-oadp-aws[A default Secret], if your backup and snapshot locations use the same credentials, or if you do not require a snapshot location.
+* An OpenShift secret that has been created from your AWS token file.
++
+[source,yaml]
+----
+cat <<EOF > ${SCRATCH}/credentials
```

Comment:
I think the velero secret must always follow this format ( aws_access_key_id= .. , aws_secret_access_key=.. )

$ cat << EOF > ./credentials-velero
[default]
aws_access_key_id=<AWS_ACCESS_KEY_ID>
aws_secret_access_key=<AWS_SECRET_ACCESS_KEY>
EOF

Above is from the official doc
https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/backup_and_restore/application-backup-and-restore#oadp-creating-default-secret_installing-oadp-aws

And the same format we use when we install OADP with ACM HUB .. 
So I am not sure the format suggested in the PR works , you want to double check with the originator ..

role_arn = .. 
web_identity_token_file = ..

Reply:
@weshayutin @mperetzred Can you advise on @birsanv's question? Thanks.

Reply:
Role_arm and web token are the fields used for STS, I don't see them further down but when the correct secrets are mounted those are the correct fields :) I hope that helps!